src/types/steps.ts

```diff
@@ -6,6 +6,9 @@ const STEP_FACE = 'face'
 const STEP_COMPLETE = 'complete'
 const STEP_AUTH = 'auth'
 const STEP_CROSS_DEVICE_SESSION_INTRO = 'crossDeviceSessionIntro'
+const STEP_PASS = 'pass'
+const STEP_REJECT = 'reject'
+const STEP_DATA = 'data'
```

Comment:
Could rename this step to STEP_DATA_CAPTURE? I think it will be more explainatory